propka/atom.py

```diff
@@ -1,66 +1,75 @@
-
-from __future__ import division
-from __future__ import print_function
-
-import string, propka.lib, propka.group
-
+"""Atom class - contains all atom information found in the PDB file"""
+import string
+import propka.lib
+import propka.group
+# TODO - what is hybrid36?
 from . import hybrid36
 
-class Atom:
-    """
-      Atom class - contains all atom information found in the pdbfile
-    """
-
-    def __init__(self, line=None, verbose=False):
 
-        self.set_properties(line)
+class Atom(object):
+    """Atom class - contains all atom information found in the PDB file"""
 
-        self.residue_label = "%-3s%4d%2s" % (self.name,self.resNumb, self.chainID)
+    def __init__(self, line=None, verbose=False):
+        """Initialize Atom object.
 
-        self.groups_extracted = 0
+        Args:
+            line:  Line from a PDB file to set properties of atom.
+            verbose:  TODO - this does not appear to be used.  Can we remove it?
+        """
+        self.occ = None
+        self.numb = None
+        self.res_name = None
+        self.type = None
+        self.chain_id = None
+        self.beta = None
+        self.icode = None
+        self.res_num = None
+        self.name = None
+        self.element = None
+        self.x = None
+        self.y = None
+        self.z = None
         self.group = None
         self.group_type = None
         self.number_of_bonded_elements = {}
-
         self.cysteine_bridge = False
-
         self.bonded_atoms = []
         self.residue = None
         self.conformation_container = None
         self.molecular_container = None
         self.is_protonated = False
-        self.steric_number_and_lone_pairs_set = False
-
+        self.steric_num_lone_pairs_set = False
         self.terminal = None
         self.charge = 0
         self.charge_set = False
         self.steric_number = 0
         self.number_of_lone_pairs = 0
         self.number_of_protons_to_add = 0
-        self.number_of_pi_electrons_in_double_and_triple_bonds = 0
-        self.number_of_pi_electrons_in_conjugate_double_and_triple_bonds = 0
+        self.num_pi_elec_2_3_bonds = 0
+        self.num_pi_elec_conj_2_3_bonds = 0
+        self.groups_extracted = 0
+        self.set_properties(line)
+        self.residue_label = "%-3s%4d%2s" % (self.name, self.res_num, self.chain_id)
 
         # ligand atom types
         self.sybyl_type = ''
         self.sybyl_assigned = False
         self.marvin_pka = False
 
-
-        return
-
-
-
-
     def set_properties(self, line):
+        """Line from PDB file to set properties of atom.
 
+        Args:
+            line:  PDB file line
+        """
         self.name = ''
         self.numb = 0
         self.x = 0.0
         self.y = 0.0
         self.z = 0.0
-        self.resNumb = 0
-        self.resName = ''
-        self.chainID = 'A'
+        self.res_num = 0
+        self.res_name = ''
+        self.chain_id = 'A'
```

Comment:
It's prettier but will it break code? – probably not... but worth keeping in mind. Definitely should merge the tests #36 first.